.env.dev

```diff
@@ -1,4 +1,5 @@
+# This loads for `yarn dev`, not the hosted dev site.
```

Comment:
Maybe changing this to ```This loads for `yarn dev` only``` (i.e., add the word "only") would make it even clearer?

Reply:
Sure